src/SampleWebApp/Controllers/AuthorssApiController.cs

```diff
@@ -0,0 +1,149 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using SampleWebApp.Data;
+using SampleWebApp.Models;
+
+namespace SampleWebApp.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/AuthorssApi")]
```

Comment:
Yes, `AuthorssApiController` was user input for the controller name. The suggested name on the UI would be AuthorsController.